Makefile

```diff
@@ -17,7 +17,7 @@ TOOLS ?= ghcr.io/siderolabs/tools:v1.4.0-alpha.0-5-g763c1d9
 PKGS ?= v1.4.0-alpha.0-11-ge561dcb
 EXTRAS ?= v1.4.0-alpha.0-2-g8cb4792
 # renovate: datasource=github-tags depName=golang/go
-GO_VERSION ?= 1.19
+GO_VERSION ?= 1.19.2
```

Comment:
This is not the real Go version used, but for some extra steps which don't use tools. I would rather keep it without a patch release

Reply:
yes, but go seems to release images for the patch versions too, or I can update the version to check for major.minor only

Reply:
updated  to only check for major.minor